packages/sdks/snippets/angular-ssr/src/app/blog-article/blog-article.component.html

```diff
@@ -0,0 +1 @@
+<p>blog-article works!</p>
```

Comment:
what is this? do we need this? can we remove it?

I want to remove any redundant file from our snippets to keep them as easy to read as possible

Reply:
Ah! the boiler plate file. I just removed it.